cloudinit/node.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is missing spaces and new lines

Reply:
hmmm I don't understand 😅

Reply:
The copyright headers seems wrong, for example https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/pkg/cloud/aws/services/userdata/node.go#L1-L15

Reply:
All the copyright headers seem to be missing the newline. It should look like:

```
Copyright...

Licensed...
```

Reply:
Ok, updated. Nothing like some bad copypasta.